html/pfappserver/root/static.alt/src/components/pfFormInputMultiple.vue

```diff
@@ -0,0 +1,300 @@
+        :options="options"
+        :taggable="true"
+        :placeholder="proxyPlaceholder"
+        :tag-placeholder="proxyTagPlaceholder"
+        @tag="addTag"
+        @change.native="onChange($event)"
+        @input.native="validate()"
+        @keyup.native.stop.prevent="onChange($event)"
+        @open="onFocus"
+        @close="onBlur"
+      ></multiselect>
+      <b-input-group-append v-if="readonly || disabled">
+        <b-button v-if="readonly || disabled" class="input-group-text" tabindex="-1" disabled><icon name="lock"></icon></b-button>
+      </b-input-group-append>
+    </b-input-group>
+    <b-form-text v-if="text" v-html="text"></b-form-text>
+  </b-form-group>
+</template>
+
+<script>
+import Multiselect from 'vue-multiselect'
+import 'vue-multiselect/dist/vue-multiselect.min.css'
+import pfMixinValidation from '@/components/pfMixinValidation'
+
+export default {
+  name: 'pf-form-input-multiple',
+  mixins: [
+    pfMixinValidation
+  ],
+  components: {
+    Multiselect
+  },
+  props: {
+    value: {
+      default: null
+    },
+    columnLabel: {
+      type: String
+    },
+    labelCols: {
+      type: Number,
+      default: 3
+    },
+    text: {
+      type: String,
+      default: null
+    },
+    disabled: {
+      type: Boolean,
+      default: false
+    },
+    readonly: {
+      type: Boolean,
+      default: false
+    },
+    separator: {
+      type: String,
+      default: ','
+    },
+    placeholder: {
+      type: String,
+      default: null
+    },
+    tagPlaceholder: {
+      type: String,
+      default: null
+    }
+  },
+  data () {
+    return {
+      focus: false
+    }
+  },
+  computed: {
+    inputValue: {
+      get () {
+        let inputValue = ((this.value)
+          ? this.value
+            .split(this.separator.trim())
+            .filter(value => value)
+            .map(value => { return { text: value.trim(), value: value.trim() } })
+          : []
+        )
+        return inputValue
+      },
+      set (newValue) {
+        this.$emit('input', newValue.map(value => value.value).filter(value => value).join(this.separator) || null)
+      }
+    },
+    options () {
+      return this.inputValue
+    },
+    forwardListeners () {
+      const { input, ...listeners } = this.$listeners
+      return listeners
+    },
+    proxyPlaceholder () {
+      return (this.focus)
+        ? this.placeholder || this.$i18n.t('Enter a new value')
+        : '' // hide placeholder when not in focus
+    },
+    proxyTagPlaceholder () {
+      return this.tagPlaceholder || this.$i18n.t('Click to add value')
+    }
+  },
+  methods: {
+    onFocus (event) {
+      this.focus = true
+    },
+    onBlur (event) {
+      this.focus = false
+    },
+    focus () {
+      this.$refs.input.focus()
+    },
+    addTag (newTag) {
+      this.$set(this, 'inputValue', [ ...this.inputValue, ...[{ text: newTag, value: newTag }] ])
+    }
+  }
+}
+</script>
+
+<style lang="scss">
```

Comment:
I propose to move common styling of components based on vue-multiselect to `src/styles/_multiselect.scss` to avoid style duplication in `pfFormInputMultiple.vue` and `pfFormChosen.vue`.